src/gui/qgsmapcanvas.h

```diff
@@ -133,6 +133,14 @@ class GUI_EXPORT QgsMapCanvas : public QGraphicsView
      */
     void setTemporalController( QgsTemporalController *controller );
 
+    /**
+     * Gets access to the temporal controller that will be used to
+     * update the canvas temporal range.
```

Comment:
Returns temporal controller

Reply:
"Gets" is consistent with other docs in the same file

Reply:
ok